qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -585,6 +588,82 @@ def _check_special_columns(cls, md_template, obj):
             cls._check_template_special_columns(md_template, obj))
 
     @classmethod
+    def _clean_validate_template(cls, md_template, study_id, obj,
```

Comment:
How do you validate that the passed study_id is actually attached to this template?